### PR TITLE
Feature/SOF-7232

### PR DESCRIPTION
**Pull request by VsevolodX**
Add materials_designer folder

### CODE REVIEW

File: other/materials_designer/definitions.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
+from IPython.display import display, Javascript
+import json
+
+
+def submit(materials):
```
Comment (timurbazhirov) [inline other/materials_designer/definitions.py:5]: (1) Maybe we should use the `get/set` naming - `get_materials` and `set_materials` here?
(2) Let's add dosctrings, pls

Comment (VsevolodX) [reply other/materials_designer/definitions.py:5]: If this is popular convention -- Ok

File: other/materials_designer/Introduction.ipynb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "\n",
+    "## Overview\n",
+    "\n",
+    "This Jupyter Lite distribution contains pre-configured examples of materials transformations including simple ones like creation of supercell or a surface, and more complex like creating an interface with Zur and McGill Superlattice matching algorithm\n",
+    "\n",
+    "## Examples\n",
+    "\n",
+    "### Zur and McGill Superlattice interface creation\n",
+    "\n",
+    "[This file](zsl_interface_generation.ipynb) helps creating interface between two materials by finding matching superlattices and selecting optimal variants by size-strain trade off.\n",
```
Comment (seankwarren) [inline other/materials_designer/Introduction.ipynb:17]: Should we cite Zur McGill paper here? [Lattice match: An application to heteroepitaxy](https://pubs.aip.org/aip/jap/article-abstract/55/2/378/502944/Lattice-match-An-application-to-heteroepitaxy?redirectedFrom=fulltext)

**Review by seankwarren — APPROVED** [review]
Make sure to include citation when notebooks are updated/finalized

File: other/materials_designer/Introduction.ipynb (hunk 2 above)
Comment (VsevolodX) [reply other/materials_designer/Introduction.ipynb]: Added DOI link

**Pull request merged**